config/config.go

```diff
@@ -38,7 +38,7 @@ func Get() (*Config, error) {
 		Brokers:                   []string{"localhost:9092"},
 		ConsumerGroup:             "dp-search-builder",
 		ConsumerTopic:             "hierarchy-built",
-		ElasticSearchAPIURL:       "http://localhost:9200",
+		ElasticSearchAPIURL:       "http://localhost:10200",
```

Comment:
Same as other PR - probably best to make sure the secrets config is explicitly setting this.